### PR TITLE
Feature/df 1659 celements and csets backlinks

### PR DESCRIPTION
**Pull request by ganqqwerty**
@emina-str @cjpsycos @birom I hope that I used the proper endpoint. Showing the backlinks for the competence sets. Only one problem now: you can click on set, but the link will be opened in the same window. Maybe we can schedule it for the improvement.

### CODE REVIEW

File: alv-portal-ui/src/app/competence-catalog/competence-elements/competence-elements.module.ts [hunk 1]
```diff
@@ -18,10 +18,10 @@ import { CompetenceElementsFilterModalComponent } from './competence-elements-fi
   ],
   declarations: [
     CompetenceElementsOverviewComponent,
-    CompetenceElementsFilterModalComponent
+    CompetenceElementsFilterModalComponent,
   ],
   entryComponents: [
-    CompetenceElementsFilterModalComponent
+    CompetenceElementsFilterModalComponent,
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/competence-catalog/competence-elements/competence-elements.module.ts]: comman needed?

File: alv-portal-ui/src/app/competence-catalog/competence-elements/competence-elements-overview/competence-elements-overview.component.html [hunk 2]
```diff
@@ -33,7 +33,9 @@ <h2 class="pr-5 mb-4 d-flex">
 
       <form [formGroup]="searchForm">
         <alv-input-field alvFormControlName="query"
-                         label="portal.competence-catalog.competence-elements.overview.search.placeholder">
+                         [attr.data-test]="'search-query'"
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/competence-catalog/competence-elements/competence-elements-overview/competence-elements-overview.component.html]: for what is "attr.data-test" needed?

File: alv-portal-ui/src/app/competence-catalog/competence-elements/competence-elements-overview/competence-elements-overview.component.ts [hunk 3]
```diff
@@ -77,4 +87,14 @@ export class CompetenceElementsOverviewComponent extends OverviewComponent<Compe
     });
   }
 
+  handleCompetenceElementActionClick(action: CompetenceCatalogAction, competenceElement: CompetenceElement) {
+    if (action === CompetenceCatalogAction.BACKLINK) {
+      this.openBacklinksModal(competenceElement);
+    }
+  }
+
+  private openBacklinksModal(competenceElement: CompetenceElement) {
```
[the remark this answers is not in the document]
Comment (emina-str) [reply alv-portal-ui/src/app/competence-catalog/competence-elements/competence-elements-overview/competence-elements-overview.component.ts]: I'm gonna fix this in other PR (for deleting). So we can merge this one.

File: alv-portal-ui/src/app/competence-catalog/competence-elements/competence-elements-overview/competence-elements-overview.component.html (hunk 2 above)
Comment (ganqqwerty) [reply alv-portal-ui/src/app/competence-catalog/competence-elements/competence-elements-overview/competence-elements-overview.component.html]: for e2e tests elements identfication

**Pull request merged**